### ISSUE TITLE
Common misspellings not marked as errors

### ISSUE THREAD

**Issue by wolf99** [streetsidesoftware/vscode-spell-checker#195]
I've noticed that, out of the box, there are common misspellings that are not found (tried with `.md` file types so far). Things such as "teh" or "hte", that are basic and very common misspellings that any spell-checker user would expect to be caught, are not.

If such misspellings are added to the `cSpell.flagWords` setting then they are caught just fine, but this step should not be necessary for such basic errors.

**Comment by will-stone** [comment]
I just came up against this and found the min word length was set to 4, so wasn't going to pick-up on problems such as `teh`. Changing `cSpell.minWordLength` to 3 solved the issue.

**Comment by jerrens** [comment]
Would it be possible for entries in flagWords to always be checked for even if they are shorter than minWordLength?  

I have many places in my source code where I take advantage of the 3-letters being ignored by spell checker (res, req, etc) and changing minWordLength to 3 means I would need to add many new words to a dictionary.  However, I would like to have common misspellings of 'the' to be marked.

Currently, when setting:
```
"cSpell.flagWords": [
    "teh"
],
```
Nothing happens

**Comment by Jason3S** [comment]
@jerrens,

This should be fix in the next release.
